src/utils/ucx/ucx_utils.h

```diff
@@ -197,8 +194,19 @@ class nixlUcxWorker {
     void reqRelease(nixlUcxReq req);
     void reqCancel(nixlUcxReq req);
 
-    /* Worker access */
-    [[nodiscard]] ucp_worker_h getWorker() const noexcept { return worker.get(); }
+    nixl_status_t
+    arm();
+
+    nixl_status_t
+    getEfd (int &fd);
+
+private:
+    /* Local UCX stuff */
```

Comment:
pls remove this comment as it provides no value

Reply:
Removed